structuredefinitions/Extension-England-ODSOrganisationRole.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-England-ODSOrganisationRole" />
+  <url value="https://fhir.nhs.uk/England/StructureDefinition/Extension-England-ODSOrganisationRole" />
+  <version value="1.1.0" />
+  <name value="ExtensionEnglandODSOrganisationRole" />
+  <title value="Extension England ODS Organisation Role" />
+  <status value="draft" />
+  <date value="2023-10-16" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="An Extension to describe what an Organisation does and whether this is the primary role of the Organisation." />
+  <purpose value="An Extension to describe what an Organization does and whether this is the primary role of the Organisation." />
+  <copyright value="Copyright © 2023+ NHS England Licensed under the Apache License, Version 2.0 (the \&quot;License\&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \&quot;AS IS\&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7® FHIR® standard Copyright © 2011+ HL7 The HL7® FHIR® standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html." />
+  <fhirVersion value="4.0.1" />
+  <kind value="complex-type" />
+  <abstract value="false" />
+  <context>
+    <type value="element" />
+    <expression value="Organization" />
+  </context>
+  <type value="Extension" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Extension">
+      <path value="Extension" />
+      <short value="An Extension to describe what an Organization does and whether this is the primary role of the Organisation." />
+    </element>
+    <element id="Extension.extension">
+      <path value="Extension.extension" />
+      <min value="5" />
+    </element>
+    <element id="Extension.extension:primaryRole">
+      <path value="Extension.extension" />
+      <sliceName value="primaryRole" />
+      <short value="This describes the type of role of the Organisation." />
+      <definition value="This denotes whether the specific role is primary or non primary. Where if the boolean value is true then this is a primary role.  Conversely, if the boolean value is false then this indicates this is a non primary role," />
```

Comment:
Remove superfluous space after full stop and before 'Conversely'. Also replace comma with full stop at end of final sentence.

Reply:
'boolean' should be 'Boolean' in all parts of the sentence.

Reply:
The second sentence would probably be better starting as 'If the Boolean value' OR 'Where the Boolean value' instead of 'Where if the boolean value'.

Reply:
OK, thanks I can make those changes.